luaui/Widgets/gfx_lupsgl4_orbs.lua

```diff
@@ -365,7 +365,7 @@ void main()
 	modelPos_vs = vec4(position.xzy*posrad.w, 0);
 	//modelPos_vs.z = fract(modelPos_vs.z * 10);
 	modelPos_vs.w = relHealth;
-	gameFrame_vs = (timeInfo.x + timeInfo.w) ;
+	gameFrame_vs = timeInfo.z;
```

Comment:
Don't forget to remove the `* 0.0333` below as well.

Reply:
the level of interactivity here is becoming unnerving :)

Reply:
https://github.com/beyond-all-reason/Beyond-All-Reason/issues/2042 relevant